api_tests/package.json

```diff
@@ -6,6 +6,7 @@
     "scripts": {
         "check": "tsc && prettier --check '**/*.{ts,json,yml}' && tslint --project .",
         "postinstall": "mkdir -p ./gen && json2ts -i ./siren.schema.json -o ./gen/siren.d.ts",
+        "pretest": "cargo build --bin cnd",
```

Comment:
I would recommend `make build` so that if we change the way we build by default, it is automatically reflected here.